### PR TITLE
Use UmbPropertyValueChangeEvent over magic string

### PR DESCRIPTION
**Pull request by enkelmedia**
Updated guide to use the new UmbPropertyValueChangeEvent-class over magic string.

## Description

_What did you add/update/change?_

## Type of suggestion

* [ ] Typo/grammar fix
* [x] Updated outdated content
* [ ] New content
* [ ] Updates related to a new version
* [ ] Other

## Deadline (if relevant)

_When should the content be published?_

### CODE REVIEW

File: 14/umbraco-cms/tutorials/creating-a-property-editor/README.md [hunk 1]
```diff
@@ -215,7 +215,7 @@ It's starting to look good! Next, let's look into setting up the event logic.
 
 Let's start with the input field. When we type something in the input field, we want the property editor's value to change to the input field's current value.
 
-We then have to dispatch an `property-value-change` event:
+We then have to dispatch an `property-value-change` event, this can be done using ´new CustomEvent('property-value-change')` or you can leverage the core class `new UmbPropertyValueChangeEvent()` (recommended):
```
Comment (alina-tincas) [inline 14/umbraco-cms/tutorials/creating-a-property-editor/README.md:218]:
```suggestion
We then have to dispatch an `property-value-change` event which can be done in two ways:
- using ´new CustomEvent('property-value-change')` or 
- using `new UmbPropertyValueChangeEvent()` which is recommended as you can leverage the core class 
```

**Comment by alina-tincas** [comment]
Hi @enkelmedia thank you for the PR 🙌 

The change looks good, however if this is changed then it needs to changed in all code examples for the whole tutorial (meaning the other articles under **creating a property editor** as the other articles build on top of the file you have made the PR for. Would you be up for making the changes to the other articles as well? 😊 

image

**Comment by enkelmedia** [comment]
@alina-tincas of course =D Sorry for not doing that from the beginning =D Made the PR from the browser yesterday.

I've updated all places where I could find the string-representation of `property-value-change`.

// m

File: 14/umbraco-cms/tutorials/creating-a-property-editor/README.md [hunk 2]
```diff
@@ -215,7 +215,9 @@ It's starting to look good! Next, let's look into setting up the event logic.
 
 Let's start with the input field. When we type something in the input field, we want the property editor's value to change to the input field's current value.
 
-We then have to dispatch an `property-value-change` event:
+We then have to dispatch an `property-value-change` event which can be done in two ways:
+- Using ´new CustomEvent('property-value-change')` or 
+- Using `new UmbPropertyValueChangeEvent()` which is recommended as you can leverage the core class
```
Comment (alina-tincas) [inline 14/umbraco-cms/tutorials/creating-a-property-editor/README.md:221]:
```suggestion
We then have to dispatch an `property-value-change` event which can be done in two ways:
- Using `new CustomEvent('property-value-change')` or 
- Using `new UmbPropertyValueChangeEvent()` which is recommended as you can leverage the core class 

1. Add the import so the event can be used:
{% code title="suggestions-property-editor-ui.element.ts" %}

import { UmbPropertyValueChangeEvent } from "@umbraco-cms/backoffice/property-editor";

{% endcode %}

2. Add the event to the property editor:

```

**Review by alina-tincas — APPROVED** [review]

**Comment by alina-tincas** [comment]
No worries, thank you for making the changes 😊 

All looks good so am merging it in 💪

**Pull request merged**